mne/event.py

```diff
@@ -885,3 +886,328 @@ def concatenate_events(events, first_samps, last_samps):
+            refEvents_t = times[refEvents_inds]
+        # adjust for trigger-stimulus delay by delaying the ref. event
+        refEvents_t += int(np.round(self._events[cat_ev]['delay'] * sfreq))
+        return refEvents_t
+
+    @property
+    def categories(self):
+        """ Return list of averaging categories in DACQ defined order. Only
+        returns categories marked active in DACQ. """
+        cats = sorted(self._categories_in_use.values(),
+                      key=lambda cat: cat['index'])
+        return cats
+
+    @property
+    def events(self):
+        """ Return events in DACQ defined order. Only returns events
+        that are in use (referred to by a category). """
+        evs = sorted(self._events_in_use.values(), key=lambda ev: ev['index'])
+        return evs
+
+    @property
+    def _categories_in_use(self):
+        return {k: v for k, v in self._categories.items() if v['state']}
+
+    @property
+    def _events_in_use(self):
+        return {k: v for k, v in self._events.items() if v['in_use']}
+
+    def get_condition(self, raw, conditions=None, stim_channel=None, mask=None,
+                      uint_cast=None, mask_type=None):
+        """ Get parameters corresponding to a category defined in
+        Elekta DACQ (data acquisition).
+        Output is designed to be used with the Epochs class to extract the
+        corresponding epochs.
+
+        Parameters
+        ----------
+        raw : Raw object
+            An instance of Raw.
+        condition : None | dict | list of dict
+            Condition or a list of conditions. Conditions can be strings
+            (DACQ comment field, e.g. 'Auditory left') or category dicts
+            (e.g. eav['Auditory left'], where eav is an instance of
+            ElektaAverager). If None, get all conditions marked active in
+            DACQ.
+        stim_channel : None | string | list of string
+            Name of the stim channel or all the stim channels
+            affected by the trigger. If None, the config variables
+            'MNE_STIM_CHANNEL', 'MNE_STIM_CHANNEL_1', 'MNE_STIM_CHANNEL_2',
+            etc. are read. If these are not found, it will fall back to
+            'STI101' or 'STI 014' if present, then fall back to the first
+            channel of type 'stim', if present.
+        mask : int | None
+            The value of the digital mask to apply to the stim channel values.
+            If None (default), no masking is performed.
+        uint_cast : bool
+            If True (default False), do a cast to ``uint16`` on the channel
+            data. This can be used to fix a bug with STI101 and STI014 in
+            Neuromag acquisition setups that use channel STI016 (channel 16
+            turns data into e.g. -32768), similar to ``mne_fix_stim14 --32``
+            in MNE-C.
+        mask_type: 'and' | 'not_and'
+            The type of operation between the mask and the trigger.
+            Choose 'and' for MNE-C masking behavior.
```

Comment:
This only defaults to `None` in `find_events` because we're in a deprecation cycle. Use `'and'` here (what it will end up being for `find_events` eventually).